### ISSUE TITLE
`pipenv --man` looks for manpage in the wrong place 

### ISSUE THREAD

**Issue by porridge** [pypa/pipenv#3690]
### Issue description

`pipenv --man` does not work

### Expected result

See the manpage.

### Actual result

```
$ LANG=C.UTF-8 pipenv --man
man: /usr/local/lib/python3.5/dist-packages/pipenv/cli/pipenv.1: No such file or directory
No manual entry for /usr/local/lib/python3.5/dist-packages/pipenv/cli/pipenv.1
```

Note: the manpage *is* installed but in `/usr/local/lib/python3.5/dist-packages/pipenv/pipenv.1`, not in `cli/`.

### Steps to replicate

Install pipenv (using pip IIRC), run `pipenv --man`.

-------------------------------------------------------------------------------
<details><summary>$ pipenv --support</summary>

Pipenv version: `'2018.11.26'`

Pipenv location: `'/usr/local/lib/python3.5/dist-packages/pipenv'`

Python location: `'/usr/bin/python3'`

Python installations found:

  - `3.5.3`: `/usr/bin/python3.5`
  - `3.5.3`: `/usr/bin/python3.5m`
  - `2.7.13`: `/usr/bin/python2`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.5.3',
 'os_name': 'posix',
 'platform_machine': 'x86_64',
 'platform_python_implementation': 'CPython',
 'platform_release': '4.19.0-0.bpo.2-amd64',
 'platform_system': 'Linux',
 'platform_version': '#1 SMP Debian 4.19.16-1~bpo9+1 (2019-02-07)',
 'python_full_version': '3.5.3',
 'python_version': '3.5',
 'sys_platform': 'linux'}
```

System environment variables:

  - `GNOME_DESKTOP_SESSION_ID`
  - `DESKTOP_AUTOSTART_ID`
  - `PWD`
  - `GDMSESSION`
  - `HOME`
  - `WINDOWPATH`
  - `PIP_SHIMS_BASE_MODULE`
  - `XDG_SESSION_ID`
  - `GPG_AGENT_INFO`
  - `QT_IM_MODULE`
  - `GDM_LANG`
  - `XDG_SEAT`
  - `_`
  - `LANG`
  - `GOBIN`
  - `DISPLAY`
  - `GTK_MODULES`
  - `QT_LINUX_ACCESSIBILITY_ALWAYS_ON`
  - `XDG_VTNR`
  - `VTE_VERSION`
  - `TERM`
  - `CLUTTER_IM_MODULE`
  - `SESSION_MANAGER`
  - `QT_QPA_PLATFORMTHEME`
  - `LOGNAME`
  - `JOURNAL_STREAM`
  - `QT4_IM_MODULE`
  - `WINDOWID`
  - `USER`
  - `XDG_SESSION_DESKTOP`
  - `LS_COLORS`
  - `PYTHONDONTWRITEBYTECODE`
  - `SHLVL`
  - `SSH_AGENT_PID`
  - `COLORTERM`
  - `XMODIFIERS`
  - `DESKTOP_SESSION`
  - `XAUTHORITY`
  - `PATH`
  - `PIP_PYTHON_PATH`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`
  - `DBUS_SESSION_BUS_ADDRESS`
  - `XDG_SESSION_TYPE`
  - `XDG_MENU_PREFIX`
  - `SSH_AUTH_SOCK`
  - `XDG_DATA_DIRS`
  - `XDG_RUNTIME_DIR`
  - `GTK_IM_MODULE`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `XDG_CURRENT_DESKTOP`
  - `USERNAME`
  - `QT_ACCESSIBILITY`
  - `SHELL`

Pipenv–specific environment variables:

Debug–specific environment variables:

  - `PATH`: `/home/porridge/.local/bin:/home/porridge/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/snap/bin:/home/porridge/go/bin:/usr/local/go/bin:/opt/idea/bin`
  - `SHELL`: `/bin/bash`
  - `LANG`: `pl_PL.UTF-8`
  - `PWD`: `/home/porridge`

---------------------------

</details>

**Comment by frostming** [comment]
This is addressed in #3246 Thanks

**Comment by porridge** [comment]
Um, but why close this report? #3246 is not merged yet, and looking at when it was created, it seems it might not be for another half a year? :-)
